doc_admin/AdminManual.adoc

```diff
@@ -1478,6 +1479,33 @@ image:image27.png[UsersListLogin.png,width=375,height=139]
 
 image:image24.png[Login.png,width=375,height=139]
 
+By default, USERSLISTLOGIN is set to no.
+
+NOTE: An application restart is required to apply the modified setting.
+
+
+==== 3.1.37 PATIENTPHOTOSTORAGE
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|Key |Default Value |Valid Values
+|PATIENTPHOTOSTORAGE |DB |DB, _<path_to_folder>_
+|===
+
+Open Hospital can save the user's profile picture in the database or on the file system.
+
+NOTE: The maximum size for patient's profile picture is limited to 4096 bytes. If a bigger image is provided it will automatically resized.
```

Comment:
```suggestion
NOTE: The maximum size for patient's profile picture is limited to 4096 bytes. If a bigger image is provided, it will automatically resized.
```